web/packages/teleterm/src/ui/AppUpdater/WidgetView.tsx

```diff
@@ -99,6 +100,13 @@ export function WidgetView({
     );
   }
 
+  if (
+    updateEvent.kind === 'error' &&
+    updateEvent.error.name === UnsupportedVersionError.name
+  ) {
+    return;
```

Comment:
I triggered this path with `TELEPORT_TOOLS_VERSION=18.1.1 TELEPORT_CDN_BASE_URL=https://cdn.cloud.gravitational.io pnpm start-term`.

What's the reason behind hiding this error in the UI? IIRC, it can only happen if I either set the env var or the cluster advertises a version of client tools that doesn't support auto-updates in Connect, right?